### PR TITLE
Update staging Github Action

### PR DESCRIPTION
**Pull request by OzzieOrca**
- Merge branch into staging if push to master, tagged PR, or updated tagged PR

### CODE REVIEW

**Review by soberstadt — CHANGES_REQUESTED** [review]
Seems like this will work.

Is this a flex that my whole merger bot could be replaced with 20 lines of Action code?

**Review by soberstadt — APPROVED** [review]
Seems like this will work.

Is this a flex that my whole merger bot could be replaced with 20 lines of Action code?

File: .github/workflows/update-staging.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [labeled, synchronize]
+
+jobs:
+  update-staging:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' || contains(github.event.pull_request.labels.*.name, 'On Staging')
```
[the remark this answers is not in the document]
Comment (OzzieOrca) [reply .github/workflows/update-staging.yml:12]: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contains

Tatai wrote the PR label merging part. I just added main merging into staging.

**Comment by OzzieOrca** [comment]
Thanks :)

Haha Github Actions is great! But don't blame me, Tatai started this. I probably never would have thought to add main merging into staging but it seemed really easy after Tatai's setup here.

**Pull request merged**